adapters/cpmstar/cpmstar_test.go

```diff
@@ -0,0 +1,11 @@
+package cpmstar
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+)
+
+func TestJsonSamples(t *testing.T) {
+	adapterstest.RunJSONBidderTest(t, "cpmstartest", NewCpmstarBidder("https://server.cpmstar.com/openrtbbidrq.aspx"))
```

Comment:
FYI: You don't need to use a real url for the tests. Consider using a simpler value like "host".